75-persistent-net-generator.rules

```diff
@@ -13,6 +13,9 @@ NAME=="?*", GOTO="persistent_net_generator_end"
 # do not create rule for eth0
 ENV{INTERFACE}=="eth0", GOTO="persistent_net_generator_end"
 
+# do not create rule for VLAN interfaces (e.g. eth1.1)
+ENV{INTERFACE}=="eth*.*", GOTO="persistent_net_generator_end"
```

Comment:
I wonder if a better way to do this might be to key on `ID_NET_DRIVER`. If it's anything other than `vif` or `ena`, we can skip custom handling.

Just  a thought; I think looking for a `.` in the interface name will work.

Reply:
Sounds good! That sounds more precise and helpful than keying in on `.`.

I'll submit the code with that for now. I can test on our machines and check back in here.

Reply:
I did test on our system and can confirm that the interface is correctly skipped.